data-validation-cataloging-pipeline/data_validation_cataloging_pipeline/stages/step_function_stage.py

```diff
@@ -0,0 +1,457 @@
+from pathlib import Path
+from typing import Any, Dict, List, Optional
+
+from aws_cdk.aws_iam import Effect, PolicyStatement
+from aws_cdk.aws_lambda import Code,  LayerVersion, Runtime, Function
+from aws_cdk.aws_stepfunctions_tasks import LambdaInvoke
+import aws_cdk.aws_stepfunctions as sfn
+from aws_ddk_core.pipelines import StateMachineStage
+from aws_ddk_core.resources import LambdaFactory
+from constructs import Construct
+import aws_cdk as cdk
+import os
+from aws_cdk.aws_kms import IKey, Key
+from aws_cdk.aws_stepfunctions import JsonPath
+from aws_cdk.aws_s3 import Bucket, IBucket
+from aws_cdk.aws_ssm import StringParameter
+from aws_cdk.aws_glue import CfnDatabase
+from aws_cdk.aws_lakeformation import CfnPermissions
+
+class StepFunctionStage(StateMachineStage):
```

Comment:
Nitpick - would be great to give this a more descriptive name describing what it does e.g. `Validation` or `Tagging`, `Cataloging` stage

Reply:
Changed name for generic StepFunctionStage to DataValidationCatalogingStage and updated all affected files

Reply:
Thanks!